rules/windows/create_remote_thread/create_remote_thread_win_susp_relevant_source_image.yml

```
@@ -62,7 +62,20 @@ detection:
             - '\winword.exe'
             - '\wmic.exe'
             - '\wscript.exe'
-    condition: selection
+    filter_explorer_vmtools:
+        SourceImage|endswith: '\explorer.exe'
+        TargetImage:
+            - 'C:\Program Files (x86)\VMware\VMware Tools\vmtoolsd.exe'
+            - 'C:\Program Files\VMware\VMware Tools\vmtoolsd.exe'
+    filter_defrag_conhost:
+        SourceImage|endswith: '\defrag.exe'
+        TargetImage: 'C:\Windows\System32\conhost.exe'
+        TargetCommandLine|contains: 'C:\WINDOWS\system32\conhost.exe 0x4'
+    filter_provtol_svchost:
+        SourceImage|endswith: '\provtool.exe'
+        SourceCommandLine|endswith: '/turn 5 /source LogonIdleTask'
+        TargetImage: 'C:\Windows\System32\svchost.exe'
+    condition: selection and not 1 of filter_*
```

As @X-Junior hopefully communicated with you. Filters should have either `optional` or `main`. Where optional represent a third party or optional feature that could be ignored in some envs, while `main` filters are most of the time builtin or necessary for the logic of the rule.

Second `TargetCommandLine` is an aurora field cannot be used as a filter (most of the time) in public rules.

Reminder this is not a Nextron repo but a public repo for all users to use.

```suggestion
    filter_optional_explorer_vmtools:
        SourceImage|endswith:
            - 'C:\Windows\explorer.exe'
            - 'C:\Windows\SysWOW64\explorer.exe'
        TargetImage:
            - 'C:\Program Files (x86)\VMware\VMware Tools\vmtoolsd.exe'
            - 'C:\Program Files\VMware\VMware Tools\vmtoolsd.exe'
    filter_main_defrag_conhost:
        SourceImage: 'C:\Windows\System32\Defrag.exe'
        TargetImage: 'C:\Windows\System32\conhost.exe'
    filter_main_provtol_svchost:
        SourceImage: 'C:\Windows\System32\provtool.exe'
        TargetImage: 'C:\Windows\System32\svchost.exe'
    condition: selection and not 1 of filter_main_* and not 1 of filter_optional_*
```